design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,165 @@
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `resource.yaml`
+
+```yaml
+apiVersion: wordpress.samples.stacks.crossplane.io/v1alpha1
+kind: WordpressInstance
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {
+        "apiVersion":"wordpress.samples.stacks.crossplane.io/v1alpha1",
+        "kind":"WordpressInstance",
+        "metadata":{
+          "annotations":{},
+          "name":"my-wordpressinstance",
+          "namespace":"app-project1-dev"
+        }
+      }
+  creationTimestamp: "2019-10-23T23:47:40Z"
+  generation: 1
+  name: my-wordpressinstance
+  namespace: app-project1-dev
+  resourceVersion: "6445"
+  selfLink: /apis/wordpress.samples.stacks.crossplane.io/v1alpha1/namespaces/app-project1-dev/wordpressinstances/my-wordpressinstance
+  uid: f2d13a15-1f9b-40a7-a173-a40abefa61bf
+  labels:
+    core.crossplane.io/parent-kind: "customresourcedefinition.apiextensions.k8s.io"
+    core.crossplane.io/parent-name: "sample-stack-wordpress"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `kubernetesapplication.yaml`
+
+```yaml
+apiVersion: workload.crossplane.io/v1alpha1
+kind: KubernetesApplication
+metadata:
+  name: wordpress-app-wordpress
+  labels:
+    stack: sample-stack-wordpress
+    core.crossplane.io/parent-kind: "wordpressinstances.wordpress.samples.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "my-wordpressinstance"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `kubernetescluster.yaml`
+
+```yaml
+apiVersion: compute.crossplane.io/v1alpha1
+kind: KubernetesCluster
+metadata:
+  name: wordpress-cluster-wordpress
+  labels:
+    stack: sample-stack-wordpress
+    core.crossplane.io/parent-kind: "wordpressinstances.wordpress.samples.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "my-wordpressinstance"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+### Example wordpress `mysqlinstance.yaml`
+
+```yaml
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstance
+metadata:
+  name: wordpress-mysql-wordpress
+  labels:
+    stack: sample-stack-wordpress
+    core.crossplane.io/parent-kind: "wordpressinstances.wordpress.samples.stacks.crossplane.io"
+    core.crossplane.io/parent-name: "my-wordpressinstance"
+    core.crossplane.io/parent-namespace: "app-project1-dev"
+    app.kubernetes.io/managed-by: stack-manager
+...
+```
+
+As per the above file example snippets, as long as the labels are defined on children CRD's,
+then we can use that label to look up all children of a given parent.
+
+Using `kubectl` for example:
+```
+kubectl get crds -l core.crossplane.io/parent-name=my-wordpressinstance
```

Comment:
This wouldn't match anything in the examples provided above, so I would recommend changing it to be one of the following instead:

```
kubectl get kubernetesapplication -l core.crossplane.io/parent-name=my-wordpressinstance
```

or

```
kubectl get crds -l core.crossplane.io/parent-name=sample-stack-wordpress
```

I would probably go with the first one since most users who are using this for debugging will probably not be looking at the crds, since they don't change very often. I guess you could always put both examples though!

Some related discussion that would influence how these queries are done can be found in #845